script.js

```diff
@@ -480,3 +479,44 @@ function sketch(p) {
     p.rect(x, y, w, h);
   }
 };
+
+if (navigator.mediaSession) {
+
+  function playAudio() {
+    var audio = document.createElement('audio');
```

Comment:
const

Reply:
oh good catch. I have added to the pr for all the good style catches. incoming shortly